src/data-workspace/data-details-sidebar/comment.js

```diff
@@ -258,14 +272,17 @@ export default function Comment({ item }) {
                 </p>
             )}
 
-            <Button
-                small
-                secondary
-                onClick={() => setEditing(true)}
-                disabled={locked}
-            >
-                {item.comment ? i18n.t('Edit comment') : i18n.t('Add comment')}
-            </Button>
+            {!canUserEditFields && (
+                <Tooltip
+                    content={i18n.t(
+                        'You do not have the authority to add or edit comments'
+                    )}
+                >
+                    {addEditButton}
+                </Tooltip>
```

Comment:
See my previous comment. Perhaps it makes sense to have a dedicated component for this. I.e. a `ButtonWithTooltip` or something.

Reply:
See note in the PR's description